arm/Microsoft.Insights/privateLinkScopes/.bicep/nested_privateEndpoint.bicep

```diff
@@ -35,7 +35,7 @@ resource privateEndpoint 'Microsoft.Network/privateEndpoints@2021-03-01' = {
     customDnsConfigs: privateEndpoint_var.customDnsConfigs
   }
 
-  resource privateDnsZoneGroups 'privateDnsZoneGroups@2020-05-01' = {
+  resource privateDnsZoneGroups 'privateDnsZoneGroups@2017-04-01' = {
```

Comment:
Was that intended?

Reply:
No. Will fix it

Reply:
Please re-run the pipeline when upating this to a new API version to be sure it works

Reply:
[![Insights: PrivateLinkScopes](https://github.com/Azure/ResourceModules/actions/workflows/ms.insights.privatelinkscopes.yml/badge.svg)](https://github.com/Azure/ResourceModules/actions/workflows/ms.insights.privatelinkscopes.yml)